cli/src/main/java/org/apache/falcon/cli/FalconEntityCLI.java

```diff
@@ -328,18 +342,18 @@ public void entityCommand(CommandLine commandLine, FalconClient client) throws I
             result = client.getDependency(entityType, entityName, doAsUser).toString();
         } else if (optionsList.contains(FalconCLIConstants.LIST_OPT)) {
             validateColo(optionsList);
-            validateEntityFields(fields);
-            validateOrderBy(orderBy, entityAction);
-            validateFilterBy(filterBy, entityAction);
+            ValidationUtil.validateEntityFields(fields);
+            ValidationUtil.validateOrderBy(orderBy, entityAction);
+            ValidationUtil.validateFilterBy(filterBy, entityAction);
             EntityList entityList = client.getEntityList(entityType, fields, nameSubsequence, tagKeywords,
                     filterBy, filterTags, orderBy, sortOrder, offset, numResults, doAsUser);
             result = entityList != null ? entityList.toString() : "No entity of type (" + entityType + ") found.";
         }  else if (optionsList.contains(FalconCLIConstants.SUMMARY_OPT)) {
-            validateEntityTypeForSummary(entityType);
+            ValidationUtil.validateEntityTypeForSummary(entityType);
             validateNotEmpty(cluster, FalconCLIConstants.CLUSTER_OPT);
```

Comment:
For the sake of consistency can we move validateNotEmpty to ValidationUtil as well?